addons/mail/static/src/core/common/composer.xml

```diff
@@ -90,7 +90,7 @@
             </div>
             <div class="o-mail-Composer-footer overflow-auto">
                 <AttachmentList
-                    t-if="allowUpload and props.composer.attachments.length > 0"
+                    t-if="props.composer.attachments.length > 0"
```

Comment:
What is the purpose of this change? I tried to revert it and all the tests in `comments_popover` are still green.

Is it an unrelated fix? The fix itself seems ok: it looks good to me that the display of attachments is not conditioned to allowing upload but to the presence of attachments instead.

But this doesn't seem explained nor covered by a test. Even more confusingly, the title of the PR/commit is "add missing allowUpload check" and the diff actually removes a check.